### PR TITLE
Update project references

### PR DESCRIPTION
**Pull request by germartinez**
Updated to be consistent with the new project names.
package-lock.json and `/dist` must be regenerated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-[![Build Status](https://travis-ci.org/gnosis/gnosis.js.svg?branch=master)](https://travis-ci.org/gnosis/gnosis.js)
+[![Build Status](https://travis-ci.org/gnosis/pm-js.svg?branch=master)](https://travis-ci.org/gnosis/pm-js)
 
 [![Slack Status](https://slack.gnosis.pm/badge.svg)](https://slack.gnosis.pm)
 
 ## Getting Started
 
-See the [documentation](https://gnosisjs.readthedocs.io/en/latest/).
+See the [documentation](https://pm-js.readthedocs.io/en/latest/).
```
Comment (germartinez) [inline README.md]: Be careful with this.

File: docs/source/events-oracles-and-markets.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ async function createDescription () {
 createDescription()
 ```
 
-Of course, future events will come to pass, and once they do, the outcome should be determinable. Oracles report on the outcome of events. The simplest oracle contract provided by Gnosis is a [`CentralizedOracle`](https://gnosis-contracts.readthedocs.io/en/latest/CentralizedOracle.html), and it is controlled by a single entity: the `owner` of the contract, which is a single Ethereum address, and which will from this point forward in this guide be referred to as the centralized oracle itself.
+Of course, future events will come to pass, and once they do, the outcome should be determinable. Oracles report on the outcome of events. The simplest oracle contract provided by Gnosis is a [`CentralizedOracle`](https://pm-contracts.readthedocs.io/en/latest/CentralizedOracle.html), and it is controlled by a single entity: the `owner` of the contract, which is a single Ethereum address, and which will from this point forward in this guide be referred to as the centralized oracle itself.
```
Comment (germartinez) [inline docs/source/events-oracles-and-markets.md]: Be careful with this.

File: docs/source/events-oracles-and-markets.md [hunk 3]
```diff
@@ -73,9 +73,9 @@ createCategoricalEvent()
 
 Note that EtherToken is traded with this particular event instance.
 
-> If you are using the Rinkeby network, you can check that your event has been indexed by [GnosisDB](https://github.com/gnosis/gnosisdb) with the following URL:
+> If you are using the Rinkeby network, you can check that your event has been indexed by [pm-trading-db](https://github.com/gnosis/pm-trading-db) with the following URL:
 > ```
-> https://gnosisdb.rinkeby.gnosis.pm/api/events/EVENT_ADDRESS
+> https://pm-trading-db.rinkeby.gnosis.pm/api/events/EVENT_ADDRESS
```
Comment (germartinez) [inline docs/source/events-oracles-and-markets.md]: Be careful with this.

**Comment by cag** [comment]
I'm gonna hold off for a moment so that all the RtD stuff can be finalized

File: package.json [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
-    "netreset-oly": "cd node_modules/@gnosis.pm/olympia-token && truffle networks --clean && node scripts/inject_network_info.js",
+    "netreset-core": "cd node_modules/@gnosis.pm/pm-contracts && truffle networks --clean && node scripts/inject_network_info.js",
+    "netreset-oly": "cd node_modules/@gnosis.pm/pm-apollo-contracts && truffle networks --clean && node scripts/inject_network_info.js",
     "nodebuild": "babel src/ --out-dir dist --source-maps",
     "webbuild": "webpack && cross-env WEBPACK_ENV=minified webpack -p",
     "dev": "run-with-testrpc 'npm run migrate && webpack-dev-server'",
     "test": "run-with-testrpc -l 40000000 'npm run migrate && npm run test-manual'",
     "test-manual": "multidep test/multidep.json && mocha --timeout 120000 --require babel-core/register $MOCHA_OPTS",
-    "test-gnosisdb": "cross-env GNOSIS_OPTIONS='{\"ipfs\":{\"host\":\"localhost\",\"port\":5001,\"protocol\":\"http\"}}' npm run test-manual",
+    "test-pm-trading-db": "cross-env GNOSIS_OPTIONS='{\"ipfs\":{\"host\":\"localhost\",\"port\":5001,\"protocol\":\"http\"}}' npm run test-manual",
```
Comment (cag) [inline package.json]: @denisgranha Gotta remember about this script name change...

File: test/test_gnosis.js [hunk 5]
```diff
@@ -92,8 +92,8 @@ describe('Gnosis', function () {
 
         await gnosis.setWeb3Provider(ganache.provider({ network_id: 4 }))
 
-        assert.equal(gnosis.olympiaToken.address, '0xa0c107db0e9194c18359d3265289239453b56cf2')
-        assert.equal(gnosis.olympiaAddressRegistry.address, '0x79da1c9ef6bf6bc64e66f8abffddc1a093e50f13')
+        assert.equal(gnosis.olympiaToken.address, '0x979861df79c7408553aaf20c01cfb3f81ccf9341')
```
Comment (cag) [inline test/test_gnosis.js]: I know these changed upstream... I can't remember if that was intentional :sweat_smile:

**Pull request merged**